epilepsy12/common_view_functions/aggregate_by.py

```diff
@@ -265,10 +265,10 @@ def calculate_kpi_value_counts_queryset(
         kpi_value_counts = KPI.objects.filter(
             registration__id__in=filtered_cases.values_list("registration")
         ).aggregate(**aggregate_queries)
-    else:
+    else: 
         kpi_value_counts = (
             KPI.objects.filter(
-                registration__id__in=filtered_cases.values_list("registration")
+                registration__id__in=filtered_cases.values("registration")
```

Comment:
Should this still be `values_list` as on line 266? I think if we change one we should change both